server/public/index.go.html

```diff
@@ -17,7 +17,39 @@
     <script src="{{ rev "/auditorium/vendor.js" }}"></script>
     <script src="{{ rev "/auditorium/index.js" }}"></script>
     <noscript>
-      {{ __ "Please enable JavaScript in your browser's settings to use this application." }}
+      <div class="f5 roboto dark-gray">
+        <div class="w-100 h3 bg-black-05">
+          <div class="mw8 center flex ph3 pt2">
+            <img src="/offen-icon-black.svg" alt="Offen logo" width="37" height="40" class="ma0 mt1 mr3">
+            <h1 class="f2 normal ma0 mt1">Offen</h1>
+          </div>
+        </div>
+        <div class="mw8 center hp0 ph3-ns">
+          <p class="dib pa2 br2 ma0 mt3 ml3 ml0-ns mr3 mr0-ns bg-light-yellow">
+            It seems like your browser is <strong>blocking JavaScript.</strong>
```

Comment:
`noscript` will be used when the browser has JavaScript disabled entirely, so maybe "has JavaScript disabled" would be the better wording. This is what others use for example: https://github.com/facebook/create-react-app/blob/f875bb0c8444720f5eac1b24822e10421b9f1202/packages/cra-template/template/public/index.html#L30